avaframe/com1DFA/com1DFA.py

```diff
@@ -298,9 +298,22 @@ def com1DFAPostprocess(simDF, tCPUDF, simDFExisting, cfgMain, dem, reportDictLis
     # this is used for the qgis connector
     cfgUtils.writeAllConfigurationInfo(avalancheDir, simDF, specDir="", csvName="latestSims.csv")
 
+    # first read all sims that are actually located in Outputs folder by checking configurationFilesDone directory
+    # this comprises all sims previously computed and still saved as well as the ones from this run
+    configDir = pathlib.Path(avalancheDir, 'Outputs', 'com1DFA', 'configurationFilesDone')
+    existingSims = list(configDir.glob('*.txt'))
+    # create a list of file names
+    existingSims = [fName.stem for fName in existingSims]
+    # create a dataframe with all simulation configurations of the ones that were actually performed
+    simDFExisting = cfgUtils.createConfigurationInfo(avalancheDir, comModule='com1DFA', standardCfg='', writeCSV=False, specDir='', simNameList=existingSims)
+
     # append new simulations configuration to old ones (if they exist),
+    # remove duplicates - sims from this run from existing
     # return total dataFrame and write it to csv
     simDFNew = pd.concat([simDF, simDFExisting], axis=0)
+    simDFNew = simDFNew.drop_duplicates(subset=['simName'])
+
+    # write the actually simulated sims to a separate csv file
     cfgUtils.writeAllConfigurationInfo(avalancheDir, simDFNew, specDir="")
```

Comment:
I suggest pushing this in a separate function

Reply:
this has been chagned